src/js/feat.js

```diff
@@ -1253,13 +1271,31 @@ const editability = {
                 return;
             }
 
+            // シフト推しながらは開店
+            if (eventN.shiftKey) {
+                data.transData = data.transData + 1;
+                if ( data.transData > 8 ){
+                  data.transData = 0;
+                }
+                component.applyUserAction(Level.A, () => {
+                  component.propagate({
+                    'transData': data.transData
+                  });
+                });
+                return;
+            }
+
             data.editing = true;
             let textareaEl;
+            let backColorEl;
+            let textColorEl;
```

Comment:
It looks like another functionality is pulled together.  I'll remove these from this PR.